diff --git a/interfaces/web/mishkal-webserver.py b/interfaces/web/mishkal-webserver.py
index 15a565a..69f3fad 100755
--- a/interfaces/web/mishkal-webserver.py
+++ b/interfaces/web/mishkal-webserver.py
@@ -20,15 +20,15 @@ def test():
     import logging
     from paste import httpserver
 
-    d=fromFs(os.path.dirname(sys.argv[0]))
-    LOG_FILENAME = os.path.join(d,u'tmp','logging_example.out')
-    logging.basicConfig(filename=LOG_FILENAME,level=logging.INFO,)
-    myLogger=logging.getLogger('MyTestWebApp')
-    h=logging.StreamHandler() # in production use WatchedFileHandler or RotatingFileHandler
-    h.setFormatter(logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s"))
-    myLogger.addHandler(h)
-    myLogger.setLevel(logging.INFO) # in production use logging.INFO
-    d=fromFs(os.path.dirname(sys.argv[0]))
+    #d=fromFs(os.path.dirname(sys.argv[0]))
+    #LOG_FILENAME = os.path.join(d,u'tmp','logging_example.out')
+    #logging.basicConfig(filename=LOG_FILENAME,level=logging.INFO,)
+    #myLogger=logging.getLogger('MyTestWebApp')
+    #h=logging.StreamHandler() # in production use WatchedFileHandler or RotatingFileHandler
+    #h.setFormatter(logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s"))
+    #myLogger.addHandler(h)
+    #myLogger.setLevel(logging.INFO) # in production use logging.INFO
+    #d=fromFs(os.path.dirname(sys.argv[0]))
     app=webApp(
       os.path.join(d,u'resources/templates'),
       staticBaseDir={u'/_files/':os.path.join(d,u'resources/files')},
